### PR TITLE
Switch to Chrome Unstable with new Headless mode

### PR DESCRIPTION
**Pull request by kripken**
After Chrome updated, we can now use unstable/dev with new headless mode and all
tests pass once more.

Previously a Chrome update caused beta to fail with either old or new headless.

### CODE REVIEW

**Comment by kripken** [comment]
I spent time trying to understand Beta's GL crashes, but I can't reproduce them locally no matter how hard I try. It seems like using swiftshader should mean we don't depend on GPU hardware, but there is still some difference that I can't pin down. Anyhow, though, it is fixed in Dev, so it seems moot.

Note that Dev also has some issue with emrun. My best guess there is something to do with the lockfile used to ensure a single instance. I saw errors on that sometimes, and deleting the file helped. emrun kills the chrome process, so that might be causing a bad lockfile situation.

**Comment by kripken** [comment]
This PR now switches us to Chrome unstable, with new headless, which seems to pass tests (all but emrun, which was disabled before).

**Review by sbc100 — APPROVED** [review]

**Comment by sbc100** [comment]
Can you update the PR description?

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,11 +42,10 @@ commands:
...
             # TODO: Make these part of the base image
             apt-get install libu2f-udev libvulkan1
-            #wget -O ~/chrome.deb https://dl.google.com/linux/direct/google-chrome-beta_current_amd64.deb
+            # wget -O ~/chrome.deb https://dl.google.com/linux/direct/google-chrome-beta_current_amd64.deb
             # If that download link breaks, temporarily use this URL instead:
-            # TODO(sbc): Disable this once we figure out whats up with the
-            # current beta.
-            wget -O ~/chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+            # wget -O ~/chrome.deb https://storage.googleapis.com/webassembly/chrome/google-chrome-stable_current_amd64.deb
+            wget -O ~/chrome.deb https://dl.google.com/linux/direct/google-chrome-unstable_current_amd64.deb
```
[the remark this answers is not in the document]
Comment (kripken) [reply .circleci/config.yml:48]: I think "unstable" and "beta" refer to the same thing.

[the remark this answers is not in the document]
Comment (sbc100) [reply .circleci/config.yml:48]: Why not uncomment line 45 instead then?

**Review by sbc100 — APPROVED** [review]
lgtm % comment

File: .circleci/config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kripken) [reply .circleci/config.yml:48]: Oops, that's wrong. Unstable==dev.

**Pull request merged**